src/meta/src/stream/source_manager/worker.rs

```diff
@@ -317,4 +334,50 @@ impl ConnectorSourceWorkerHandle {
 
         Ok(discovered_splits)
     }
+
+    fn send_command(&self, command: SourceWorkerCommand) -> MetaResult<()> {
+        let cmd_str = format!("{:?}", command);
+        self.command_tx
+            .send(command)
+            .with_context(|| format!("failed to send {cmd_str} command to source worker"))?;
+        Ok(())
+    }
+
+    /// Force [`ConnectorSourceWorker::tick()`] to be called.
+    pub async fn force_tick(&self) -> MetaResult<()> {
+        let (tx, rx) = oneshot::channel();
+        self.send_command(SourceWorkerCommand::Tick(tx))?;
+        rx.await
+            .context("failed to receive tick command response from source worker")?
+            .context("source worker tick failed")?;
+        Ok(())
+    }
```

Comment:
I don't fully agree with making `force_tick` rely on the command channel. 
In prev impl, each enumerator's `force_tick` is independent with each other. But in this impl, each request to `force_tick` must be handled sequentially.

Consider this case, a channel containing `(drop_fragment(x), force_tick(y))`, when handling `drop_fragment(x)`, the network to x's broker is unstable and the call may hang for a while, or forever. But it blocks call for `force_tick(y)`.
A real-world case, we call `force_tick` when creating a source, the behavior to the user is like everything is good but SQL never returns, though the root cause is irrelevant to the creating one. It can cause much trouble when doing POC.

Reply:
I don't get your point. The new impl is exactly the same as before

Reply:
> A real-world case, we call `force_tick` when creating a source

No it will only be called in rare cases when `splits` is `None`. When `CREATE SOURCE`, we call `create_source_worker`, which will `tick` before creating the source worker

Reply:
I don't think it's a big deal to handle `force_tick` separately here. If `drop_fragment` hangs, `tick` will also unlikely succeed

Reply:
> If drop_fragment hangs, tick will also unlikely succeed

I don't seem to understand your logic here. Drop_fragment and force_tick can happen to different sources, why they tend to fail together?

Reply:
1 ConnectorSourceWorker corresponds to 1 SplitEnumerator. 

> In prev impl, each enumerator's force_tick is independent with each other

enumerators are still independent with each other now

Reply:
1 source - 1 split enumerator - 1 source worker - 1 channel for drop_fragment & force_tick

So they don't "happen to different sources" :eyes:

Reply:
My mistake, the command channel is built inside each worker rather than the manager.